.github/workflows/ci.yaml

```diff
@@ -10,6 +10,11 @@ on:
       - "ubuntu-*"
 
 jobs:
+  cla-check:
+    if: github.event_name == 'pull_request'
```

Comment:
I am wondering if we should have added this condition inside the reusable workflow in `main`. It's no biggie though, so let's just go with it.

Reply:
hum could be indeed. although, wouldn't it be `workflow_call` then?